### PR TITLE
Fix: #705 Now folders can have space only titles

### PR DESCRIPTION
**Pull request by ViktorSimko**

### CODE REVIEW

**Comment by jhreis** [comment]
May not be necessary to merge, but would be nice to gets some automated tests in for these folder titles at some point (creating and updating).

File: Client/Frontend/Home/BookmarksPanel.swift [hunk 1]
```diff
@@ -209,6 +218,9 @@ class BookmarkEditingViewController: FormViewController {
                 row.title = Strings.Name
                 row.value = bookmark.title
                 self.titleRow = row
+            }.cellSetup { cell, row in
+                cell.textField.addTarget(EurekaTextRowHelper.self, action: #selector(EurekaTextRowHelper.replaceNormalSpacesWithNonBreakingSpaces(_:)), forControlEvents: .EditingChanged)
+                cell.textField.addTarget(EurekaTextRowHelper.self, action: #selector(EurekaTextRowHelper.replaceNonBreakingSpacesWithNormalSpaces(_:)), forControlEvents: .EditingDidEnd)
```
Comment (garvankeeley) [inline Client/Frontend/Home/BookmarksPanel.swift]: Hi I'm not familiar enough with the BM editing code. This means: during editing, a space gets replaced with an nbsp and then when completed gets converted back? Why is this needed? Thanks

Comment (ViktorSimko) [inline Client/Frontend/Home/BookmarksPanel.swift]: Hi 

Yes. 
I think it's needed because spaces doesn't appear at the end of the text in right-aligned text fields.
And also when you try to rename a folder to a space only name and delete the old name, then if you type in a space the cursor jumps to the left side of the text row.

Comment (garvankeeley) [inline Client/Frontend/Home/BookmarksPanel.swift]: ok thanks!
